### ISSUE TITLE
Expand use of failOnFailedSeed option

### ISSUE THREAD

**Issue by ldko** [webrecorder/browsertrix-crawler#360]
I was looking at where `--failOnFailedSeed` was added in #300 and am wondering if it would generally make sense (it does in my mind for my use case) for that to also apply when a [seed isn't parsed successfully](https://github.com/webrecorder/browsertrix-crawler/blob/cf404efa13b879a770b789ecbb86fb4bec2d6e1c/util/seeds.js#L10). Right now if I crawl with a seed list (generated automatically from URLs extracted from a document) containing this particular URL https://doi.org10.1901/jaba.2012.45-85 that is missing a slash after the .org and is presumably read as having a TLD of .1901, the crawl aborts. Perhaps if `failOnFailedSeed` is false, such a problem could be logged lower than fatal, so the crawl doesn't abort?

**Comment by tw4l** [comment]
Perhaps we could have the following behavior for seeds that aren't parsed successfully:

- If it's the only seed or `--failOnFailedSeed` is set, fail the crawl
- If there are multiple seeds, not all of them are invalid, and `--failOnFailedSeed` is not set, log an error and continue the crawl

**Comment by tw4l** [comment]
In simpler terms, if `--failOnFailedSeed` is not set, we'll want to check that we have at least one valid seed and continue on if so or fail if not.

**Comment by ldko** [comment]
That logic makes sense to me. :)
